realm/realm-annotations-processor/src/main/java/io/realm/processor/RealmProxyClassGenerator.java

```diff
@@ -495,6 +534,33 @@ private void emitInjectContextMethod(JavaWriter writer) throws IOException {
         writer.emitEmptyLine();
     }
 
+    private void emitBacklinkFieldAccessors(JavaWriter writer) throws IOException {
+        for (Backlink backlink : metadata.getBacklinkFields()) {
+            String cacheFieldName = backlink.getTargetField() + BACKLINKS_FIELD_EXTENSION;
+            String realmResultsType = "RealmResults<" + backlink.getSourceClass() + ">";
+
+            // Getter
+            writer.beginMethod(realmResultsType, metadata.getInternalGetter(backlink.getTargetField()), EnumSet.of(Modifier.PUBLIC));
```

Comment:
Yes.  The lack of consistency bothers me much more than the noise.  Will restore to original writerless where possible.